### DISCUSSION TITLE
Welcome to curator-opensearch Discussions!

### DISCUSSION THREAD

**Discussion by uzhinskiy** [flant/curator-opensearch#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eyalc10** [comment]
I have opened a ticket with the restore functionality and would be happy for a review of the solution I have provided
https://github.com/flant/curator-opensearch/issues/29
